Project3_STL/headers/Triangle.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include "Point3D.h"
+
+namespace Shapes3D
+{
+    class Triangle1
+    {
+    public:
+        Triangle1();
+        Triangle1(Point3D p1, Point3D p2, Point3D p3);

```

Comment:
IDK but does it work without specifying namespace in parameters? It should Shapes3D::Point3D p1 and so on.

Reply:
> IDK but does it work without specifying namespace in parameters? It should Shapes3D::Point3D p1 and so on.

The Triangle class in mentioned within the namespace so it'll work fine.
namespace Shapes3D
{
class Triangle 
{
-----
}

}